pages/audit/rules.mdx

```diff
@@ -0,0 +1,119 @@
+---
+layout: 'guide'
+title: 'Avo tracking plan audit rules'
+abstract: 'Learn how to read and fix your tracking plan audit'
+---
+
+import Link from '../../components/Link';
+
+# Audit rules
+
+Following are the static rules Avo tests for in the tracking plan audit. To learn more about how to get started with the tracking plan audit, see <Link passHref href="/audit/quickstart"><a>Quickstart: Audit in 5 minutes</a></Link>.
```

Comment:
Should Quickstart: Audit in 5 minutes be the actual title of the `audit/quickstart.mdx` doc?

Reply:
Ah good catch, it was, then I renamed it, will update it here

Reply:
Done in 9f31e1575cd0a6d672211a1dbb9ee81c71692de7